language-plutus-core/prelude/PlutusPrelude.hs

```diff
@@ -170,6 +171,11 @@ prettyStringBy = docString .* prettyBy
 prettyTextBy :: PrettyBy config a => config -> a -> T.Text
 prettyTextBy = docText .* prettyBy
 
+-- | Fold a monadic function over a 'Foldable'. The monadic version of 'foldMap'.
+foldMapM :: (Foldable f, Monad m, Monoid b) => (a -> m b) -> f a -> m b
```

Comment:
I thought I need this, but it turned out I do not need it. But it's a common thing, so I decided to keep it.